src/Packagist/Api/Client.php

```diff
@@ -187,6 +189,88 @@ public function popular(int $total): array
         return array_slice($results, 0, $total);
     }
 
+    /**
+     * Get a list of known security vulnerability advisories
+     *
+     * @param array $packages
```

Comment:
My initial instinct when testing this was to call `$client->advisories('monolog/monolog')`. I think it's fine to enforce an array of package names here (maybe add this as a description on this argument). I then thought it might be nice to be able to access advisories from the Package class, i.e. `$package->advisories()`, but the Package class doesn't have client awareness so it would require making the AbstractResult classes aware of the client. I don't think it's worth it, but it would be nice to have that accessor method.

(Not asking for changes here, just voicing my thoughts while testing)

Reply:
Yeah I thought about allowing `Package` values in the array, but all of the other methods that accept a package as an argument only accept the string names, so I figured it made sense to be consistent and do the same here.

Maybe a future enhancement could be to change all of those to accept `Package|string`.